test/unit/archipelago-controller.spec.ts

```diff
@@ -1,206 +0,0 @@
-import expect from 'assert'
```

Comment:
This test is been removed because it's not really that useful and a lot of things were implemented just to support this test (like changing the current internals options of archipelago on the fly)